loadtest/loadtest.go

```diff
@@ -158,9 +164,11 @@ func New(config *config.LoadTestConfig, nc NewController) *LoadTester {
 	if config == nil || nc == nil {
 		return nil
 	}
+	id, _ := uuid.NewV1()
```

Comment:
I would probably keep the Id external. Only the API layer needs this.

Reply:
satori/uuid is unmaintained. We should use https://github.com/gofrs/uuid now. Also, let's use V4 instead of V1.